ui-library-filesharing-chat-composite/api/UploadFileToAzureBlobStore/index.ts

```diff
@@ -0,0 +1,86 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+import { AzureFunction, Context, HttpRequest } from '@azure/functions';
+import * as multipart from 'parse-multipart';
+import HTTP_CODES from 'http-status-enum';
+
+const azureStorageConnectionString = process.env['azureStorageConnectionString'];
+let storageAccountName;
+try {
+  console.log("****"+azureStorageConnectionString);
+  storageAccountName = azureStorageConnectionString.match(/AccountName=([^;]+)/)[1];
+} catch {
+  console.warn('Azure storage connection string not provided');
+}
+const fileSharingUploadsContainerName = 'uploads'; // Must be in storage path value in function.json
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+const httpTrigger: AzureFunction = async function (context: Context, req: HttpRequest): Promise<any> {
+  context.log('HTTP trigger function processed a request to upload a file to azure blob store.');
+
+  if (!req.body || !req.body.length) {
+    context.res.body = { error: `Request body is not defined` };
+    context.res.status = HTTP_CODES.BAD_REQUEST;
+    return;
+  }
+
+  // `filename` is required property to use multi-part npm package
```

Comment:
This comment sounds a bit wrong to me? multi-part has to do with chunking data in an HTTP request. The filename should be unrelated to that? We need the filename so that we can name the blob being uploaded to azure storage.